logging/src/main/java/org/jboss/as/logging/logmanager/WildFlyLogContextSelectorImpl.java

```diff
@@ -51,10 +52,16 @@ public LogContext getLogContext() {
 
     WildFlyLogContextSelectorImpl(final LogContextSelector defaultLogContextSelector) {
         // There is not a way to reset the LogContextSelector after a reload. If the current selector is already a
-        // WildFlyLogContextSelectorImpl we should use the system default selector. This avoids possibly wrapping the
+        // WildFlyLogContextSelectorImpl we should use the previous default selector. This avoids possibly wrapping the
         // same log context several times. It should also work with the embedded CLI selector as the commands handle
         // setting and resetting the contexts.
-        final LogContextSelector dft = (defaultLogContextSelector instanceof WildFlyLogContextSelectorImpl ? LogContext.DEFAULT_LOG_CONTEXT_SELECTOR : defaultLogContextSelector);
+        final LogContextSelector dft;
+        if (defaultLogContextSelector instanceof WildFlyLogContextSelector) {
```

Comment:
Nit: you check for instanceof the interface but then cast to the concrete impl.

Reply:
Whoops yeah. I had a getter at one pint then decided it was a bad idea and forgot to change the case. I'll fix that.